code/datums/martial/krav_maga.dm

```diff
@@ -0,0 +1,211 @@
+/datum/martial_art/krav_maga
+	name = "Krav Maga"
+	counter_prob = 25
+
+/datum/martial_art/krav_maga/on_hit(var/mob/living/carbon/human/A, var/mob/living/carbon/human/D)
+	if(prob(counter_prob))
+		if(prob(50))
+			A.visible_message("<span class='warning'>[A] counters [D]'s hit!</span>", \
+						 	"<span class='userdanger'>You counter the hit!</span>")
+			sleep(5)
+			playsound(get_turf(A), 'sound/effects/hit_block.ogg', 50, 1, -1)
+			D.apply_damage(10, BRUTE)
+			return 1
+
+		else
+			A.visible_message("<span class='warning'>[A] blocks [D]'s hit!</span>", \
+						 	"<span class='userdanger'>You block the hit!</span>")
+			playsound(get_turf(A), 'sound/effects/hit_block.ogg', 50, 1, -1)
+			return 1
+	return 0
+
+
+/datum/martial_art/krav_maga/teach(var/mob/living/carbon/human/H,var/make_temporary=0)
+	..()
+	H << "<span class = 'userdanger'>You know the arts of Krav Maga!</span>"
+	H << "<span class = 'danger'>Recall your teachings using the Access Tutorial verb in the Krav Maga menu, in your verbs menu.</span>"
+	H.verbs += /mob/living/carbon/human/proc/krav_maga_help
+	H.verbs += /mob/living/carbon/human/proc/neck_chop
+	H.verbs += /mob/living/carbon/human/proc/head_elbow
+	H.verbs += /mob/living/carbon/human/proc/leg_sweep
+	H.verbs += /mob/living/carbon/human/proc/quick_choke
```

Comment:
these all need to be spells or something---I dunno. Verbs are gross; action buttons are better.